spec/rubocop/cop/performance/big_decimal_string_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Performance::BigDecimalString do
+  subject(:cop) { described_class.new }
+
+  it 'registers an offense and corrects when using `BigDecimal` with integer' do
+    expect_offense(<<~RUBY)
+      BigDecimal(1)
+      ^^^^^^^^^^^^^ Convert numeric argument to string before passing to `BigDecimal`.
+    RUBY
+  end
+
+  it 'registers an offense and corrects when using `BigDecimal` with float' do
+    expect_offense(<<~RUBY)
+      BigDecimal(1.5, 2, exception: true)
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Convert numeric argument to string before passing to `BigDecimal`.
```

Comment:
It would be better if the warning range could be specified.

```suggestion
      BigDecimal(1.5, 2, exception: true)
                 ^^^ Convert numeric argument to string before passing to `BigDecimal`.
```